src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
-   :transactions-confirmed                {:one   "Transaction confirmée"
-                                           :other "{{count}} transactions confirmées"
-                                           :zero  "Aucune transaction confirmée"}
    :transaction                           "Transaction"
-   :unsigned-transactions                 "Transactions non signées"
-   :no-unsigned-transactions              "Aucune transaction non signée"
-   :enter-password-transactions           {:one   "Confirmez la transaction en saisissant votre mot de passe"
-                                           :other "Confirmez les transactions en saisissant votre mot de passe"}
+   :unsigned-transaction-expired          "La transaction non signée a expiré"
    :status                                "Statut"
-   :pending-confirmation                  "Confirmation en attente"
    :recipient                             "Destinataire"
-   :one-more-item                         "Encore un élément"
-   :fee                                   "Frais"
-   :estimated-fee                         "Frais Estimés"
-   :value                                 "Valeur"
    :to                                    "à"
    :from                                  "de"
    :data                                  "Données"
    :got-it                                "Compris"
-   :contract-creation                     "Création du contrat"
+   :block                                 "Bloc"
+   :hash                                  "Hash"
+   :gas-limit                             "limite Gas"
+   :gas-price                             "prix Gas"
+   :gas-used                              "Gas utilisé"
+   :cost-fee                              "Coût/Frais"
+   :nonce                                 "Nonce"
+   :confirmations                         "Confirmations"
+   :confirmations-helper-text             "Veuillez attendre au moins 12 confirmations pour être sûr que votre transaction a bien été définitivement validée"
+   :copy-transaction-hash                 "Copier le hash de la transaction"
+   :open-on-etherscan                     "Ouvrir sur Etherscan.io"
 
    ;:webview
    :web-view-error                        "Oops, erreur"})
+   
+   ;;testfairy warning
+   :testfairy-title                       "Avertissement!"
+   :testfairy-message                     "Vous utilisez une application installée à partir d'une version de développement. À des fins de test, cette version inclut l'enregistrement de session si la connexion Wi-Fi est utilisée. Toutes vos interactions avec cette application sont sauvegardées (vidéo et logs) et pourraient être utilisées par notre équipe de développement pour enquêter sur d'éventuels problèmes. Les vidéos / logs enregistrés n'incluent pas vos mots de passe. L'enregistrement est effectué uniquement si l'application est installée à partir d'une version instable. Rien n'est enregistré si l'application est installée depuis PlayStore ou TestFlight."
```

Comment:
put "(nightly builds)" after "version instable" to make it clear what a "version instable" is

Reply:
got it.